src/plugins/auto/src/auto_schedule.cpp

```diff
@@ -883,23 +884,29 @@ AutoSchedule::~AutoSchedule() {
                     LOG_INFO_TAG("CPU_HELP:fps:%lf", count * 1000 / durtation.count());
                 }
             } else {
-                LOG_INFO_TAG("%s:infer:%ld", _workerRequest.first.c_str(), count);
-                auto n = reqAllStartTimes.size();
                 Time time;
+                // skip the first infer, Calculate fps using the time between the last infer and the second infer
+                // keep count as real count, don't count-- when skipping the first infer
+                if (!reqAllStartTimes.empty()) {
+                    reqAllStartTimes.pop_front();
+                    statis_count--;
+                }
                 while (!reqAllStartTimes.empty()) {
                     time = reqAllStartTimes.front();
                     if (time < _cpuHelpReleaseTime) {
                         reqAllStartTimes.pop_front();
-                        n--;
+                        statis_count--;
```

Comment:
from my opinion, it seems variable `statis_count ` have no use except for  `statis_count--` operation within this code block. is this the expected logic?